include/irrUString.h

```diff
@@ -886,10 +853,7 @@ class ustring16
 		for (; first != last; ++first)
 			append((uchar32_t)*first);
 	}
-#endif
 
-
-#ifndef USTRING_CPP0X_NEWLITERALS
```

Comment:
If you remove <i>exactly</i> the function inside `#ifndef USTRING_CPP0X_NEWLITERALS` then it should work